shell/platform/android/io/flutter/plugin/platform/VirtualDisplayController.java

```diff
@@ -128,6 +129,16 @@ public int getRenderTargetHeight() {
   }
 
   public void resize(final int width, final int height, final Runnable onNewSizeFrameAvailable) {
+    // When 'hot reload', although the resize method is triggered, the size of the native View has
+    // not changed.
+    if (width == getRenderTargetWidth() && height == getRenderTargetHeight()) {
+      getView().postDelayed(onNewSizeFrameAvailable, 0);
```

Comment:
What is the contract around onNewSizeFrameAvailable? In the case that the size doesn't change do we need to run it?

Reply:
yes, `postDelayed` can be removed

Reply:
I don't think this is true, it must be called.